src/main/java/io/supertokens/config/annotations/DifferentAcrossTenants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *    Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+ package io.supertokens.config.annotations;
+
+ import java.lang.annotation.ElementType;
```

Comment:
We might not need this annotation, instead we can infer - Is JsonProperty and does not have ConfigYamlOnly and NotConflictingInApp annotations. Cross check if this is correct

Reply:
Yeah you are right removed this